src/js/config/Categories.js

```diff
@@ -66,6 +66,9 @@ var Categories = [
         ops: [
             "AES Encrypt",
             "AES Decrypt",
+            "Affine Cipher Encode",
```

Comment:
I'd probably move these down below the XOR operations. This isn't supposed to be an alphabetical list and these operations will be used more rarely.